tests/it/main.rs

```diff
@@ -40,11 +49,23 @@ async fn create_simple_table(client: &Client, table_name: &str) {
         .unwrap();
 }
 
-async fn fetch_simple_rows(client: &Client, table_name: &str) -> Vec<SimpleRow> {
+async fn create_rename_table(client: &Client, table_name: &str) {
+    client
+        .query("CREATE TABLE ?(fixId UInt64, extComplexId String, extFloat Float64) ENGINE = MergeTree ORDER BY fixId")
+        .bind(Identifier(table_name))
+        .execute()
+        .await
+        .unwrap();
+}
```

Comment:
The functions related to the "simple rows" with a few fields were put in main.rs because it's useful for many different tests. I'd say renaming is a specific case and the related things (table creation, struct definition) should be kept in the related test file instead.